projects/common/lib/components/checkbox/checkbox.component.ts

```diff
@@ -48,17 +46,28 @@ export class CheckboxComponent extends Base implements  ControlValueAccessor {
   public _onChange = (_: any) => {};
   public _onTouched = () => {};
 
+  constructor( @Optional() @Self() public controlDir: NgControl ) {
+    super();
+    if ( controlDir ) {
+      controlDir.valueAccessor = this;
+    }
+  }
 
-  constructor() { super(); }
-
+  ngOnInit() {
+    if ( this.errorMessageRequired ) {
+      this.defaultErrorMessage = this.errorMessageRequired;
+    } else {
+      this.defaultErrorMessage = this.label + ' is required.';
+    }
+  }
 
-  /*setCheckboxVal(event: boolean) {
+  setCheckboxVal(event: boolean) {
     console.log(event);
```

Comment:
I'll do this myself, but let's remove the console log in situations like that.  As a rule of thumb, I do not like console logs in our builds of the library, especially if they're just debug noise. 

(To be clear, I've done this so don't worry about it, I'm just letting you know)